cpp/src/arrow/compute/exec/options.h

```diff
@@ -304,10 +310,9 @@ class ARROW_EXPORT TableSinkNodeOptions : public ExecNodeOptions {
  public:
   TableSinkNodeOptions(std::shared_ptr<Table>* output_table,
                        std::shared_ptr<Schema> output_schema)
```

Comment:
We can remove this second argument now right?

Reply:
Good catch.  I've removed it.